hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/validation/RequestValidations.java

```diff
@@ -56,15 +66,17 @@ public RequestValidations withinContext(ValidationContext validationContext) {
   }
 
   public synchronized RequestValidations load() {
-    registry = new ValidatorRegistry(validationsPackageName);
+    registry = new ValidatorRegistry<>(Type.class, validationsPackageName,
+        Arrays.stream(VersionExtractor.values()).map(VersionExtractor::getValidatorClass).collect(Collectors.toSet()),
+        ALLOWED_REQUEST_PROCESSING_PHASES);
     return this;
   }
 
   public OMRequest validateRequest(OMRequest request)
       throws Exception {
-    List<Method> validations = registry.validationsFor(
-        conditions(request), request.getCmdType(), PRE_PROCESS);
 
+    List<Method> validations = registry.validationsFor(request.getCmdType(), PRE_PROCESS,
+        this.getVersions(request));
```

Comment:
nit: remove `this.` and do not wrap line.

Reply:
done